app_dart/lib/src/request_handlers/github/webhook_subscription.dart

```diff
@@ -171,8 +172,16 @@ class GithubWebhookSubscription extends SubscriptionHandler {
       final RepositorySlug slug = pullRequestEvent.repository!.slug();
       final GitHub gitHubClient = await config.createGitHubClient(pullRequest: pr);
       final String body = config.mergeConflictPullRequestMessage;
-      if (!await _alreadyCommented(gitHubClient, pr, body)) {
-        await gitHubClient.issues.createComment(slug, pr.number!, body);
+      if (!await _alreadyCommented(
+        gitHubClient,
+        pr,
+        body,
+      )) {
+        await gitHubClient.issues.createComment(
+          slug,
+          pr.number!,
+          body,
```

Comment:
Generally, if a method is all positional args, you avoid the trailing comma to make it easier to read

Reply:
Fixed.